lib/wrappers/rust/Cargo.toml

```diff
@@ -7,5 +7,6 @@ edition = "2021"
 
 [dependencies]
 contourwall_core = {path = "../../cw-core" , version = "0.1.0" }
```

Comment:
Replace this line `contourwall_core = {path = "../../cw-core" , version = "0.1.0" }` with the following:

```toml
# Uncomment the following line if you are developing the wrapper and want the newest version of the core libary
# contourwall_core = {path = "../../cw-core" }
contourwall_core = "0.2.1"
```

Tomorrow I am publishing a newer version of the core libary (probably after this branch has been merged), this will reference to the newest **tested** version, instead of a version which is still in development.